### ISSUE TITLE
logging error with 1.2.0 version

### ISSUE THREAD

**Issue by psj8252** [PyTorchLightning/pytorch-lightning#6172]
## 🐛 Bug

if I use logger named "lightning", logger print log twice.
It happens with pytorch-lightning  1.2.0 version and doesn't happend with 1.1.8 version. 
This is reproduced easily like below.

Error Example 
```python
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytorch_lightning as pl
>>> import logging
>>> logger = logging.getLogger("lightning")
>>> logger.info("test")
test
>>> logging.basicConfig(level=logging.INFO)
>>> logger.info("test")
test
INFO:lightning:test
>>> pl.__version__
'1.2.0'
```
```python
>>> import pytorch_lightning as pl
>>> import logging
>>> logger = logging.getLogger("lightning")
>>> logger.info("test")
>>> logging.basicConfig(level=logging.INFO)
>>> logger.info("test")
INFO:lightning:test
>>> pl.__version__
'1.1.8'
```

## Please reproduce using the BoringModel

Don't need model.

### To Reproduce

Refer above codes 1.2.0 version

### Expected behavior

Refer above codes 1.1.8 version

### Environment

 - PyTorch Version (e.g., 1.0): 1.2.0
 - OS (e.g., Linux): Ubuntu 20.04.1 LTS
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.7,9
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by awaelchli** [comment]
Hi, you need to use 

`logger = logging.getLogger("pytorch_lightning")`

instead of 

`logger = logging.getLogger("lightning")`

then you will get the correct logger. 

Apologies, the docs here seem to be outdated
https://pytorch-lightning.readthedocs.io/en/latest/extensions/logging.html#configure-console-logging
I will update that

**Comment by awaelchli** [comment]
@psj8252 We merged a fix that was in 1.1.6 - 1.1.8 but it got lost and didn't make it into 1.2. 
After Wednesday when the new version is out you can upgrade Lightning and duplicate logging will not occur anymore.
See the updated documentation section:
https://pytorch-lightning.readthedocs.io/en/latest/extensions/logging.html#configure-console-logging
